gpytorch/__init__.py

```diff
@@ -39,6 +40,9 @@
 # Old deprecated stuff
 fast_pred_var = beta_features._moved_beta_feature(settings.fast_pred_var, "gpytorch.settings.fast_pred_var")
 
+# Get the version
+with open(os.path.join(os.path.dirname(__file__), "..", "VERSION"), "r") as f:
```

Comment:
So doing this kind of stuff outside the `setup.py` file can bey annoying in the context of some build systems (e.g. ours...) that repackage the source into .par files or the like. This can be handled by explicit specifying the source files in the build system so it's workable, but I'd advise you to use the `os` business sparingly -  otherwise it'll make it harder to use the package in non-vanilla environments.

An alternative would be to just have a standard `version.py` python module that specifies a single `VERSION` variable - that kind of stuff is typically handled automatically by build systems. Or just specify that variable directly in `setup.py` if that't the only place where it is used.

Reply:
Got rid of the os stuff.